api/osrd_infra/models/physical_approximations.py

```diff
@@ -0,0 +1,11 @@
+from django.contrib.gis.db import models
+from osrd_infra.utils import PydanticValidator
+
+from osrd_infra.schemas.physical_approximations import ElectricProfilesList as ElectricProfilesListSchema
+
+
+class ElectricProfilesList(models.Model):
```

Comment:
I'm not a big fan of the name.

- I would remove `List`
- The notion of voltage drop does not appear in this name

Reply:
* I changed `List` to `Set`, because I feel it is important we stress that this model is a package of several objects at once
* I agree, but  what the class represents is actually closer to "approximate resistance per class", and that makes a more complicated name than electrical profile... I think it's not so bad